lib/board.rb

```diff
@@ -34,8 +34,48 @@ def free_positions
     board.flatten.select { |cell| cell.is_a?(Integer) }
   end
 
+  def string_to_board(board_string)
```

Comment:
+1, this method doesn't really belong here, does it?
Sometimes it's okay to have methods in the production code to aid testing or increase testability. In this case though it feels like a better place would be a helper method in the test file I think.